homework/06LessonSix.js

```diff
@@ -76,28 +81,32 @@ function ConferenceRoom() {
 // Add a method called sayHi() that returns "Hi, I'm " + this.name + '!';
 function Person(name) {
   this.name = name;
+  this.sayHi = function() {
+    return "Hi, I'm " + this.name + "!";
+  }
+  
 }
 
 // Problem 5:
 // create a new Person named Jen
-var jen;
+var jen = new Person("Jen");
 
 // Problem 6:
 // add jen to the conferenceRoom
 var conferenceRoom = new ConferenceRoom();
 // you can do this by calling the method enter() as follows:
 // conferenceRoom.enter(somePerson);
 // add your code for Problem 6 here
-
+conferenceRoom.enter(jen);
 
 // ****
 // Concept Checkpoint
 //
 // Write your answer in comments
 //
 // What is a method?
-//
-//
+// A method is a function that contains a function definition. It is used to add definitions 
```

Comment:
A method is a _property_ that contains a function definition.

It's a function that is assigned to a object as a object property.